### PR TITLE
Update office-365-user-email-settings.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a new para about how to access the Mail pane.

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9404

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/email/office-365-user-email-settings.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ description: "This article gives you information on managing settings for your u
 
 As the admin of an organization, there are email settings you can manage on your users. This article gives you information on managing these settings.
 
+## Manage email settings
+
+1. Go to the admin center at <https://admin.microsoft.com>.
+2. Go to **Users** > **Active users**, and select the user for whom you would like to manage email settings.
+3. In the new pane, click on **Mail** to manage email settigs.
+
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/office-365-user-email-settings.md]:
```suggestion
2. Go to **Users** > **Active users** and select the user for whom you would like to manage email settings.
3. In the new pane, select **Mail** to manage email settigs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/email/office-365-user-email-settings.md [hunk 2]
```diff
@@ -23,6 +23,12 @@ description: "This article gives you information on managing settings for your u
 
 As the admin of an organization, there are email settings you can manage on your users. This article gives you information on managing these settings.
 
+## Manage email settings
+
+1. Go to the admin center at <https://admin.microsoft.com>.
+2. Go to **Users** > **Active users** and select the user for whom you would like to manage email settings.
+3. In the new pane, select **Mail** to manage email settigs.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/office-365-user-email-settings.md]:
```suggestion
3. In the new pane, select **Mail** to manage email settings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@msbemba, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako

**Pull request merged**